app/uk/gov/hmrc/payetaxcalculatorfrontend/views/main_template.scala.html

```diff
@@ -23,6 +23,15 @@
     }
 }
 
+@scriptElems = {
```

Comment:
if this is google analytics script why don't we call it appropriately. Now we have bot `scriptElem` (singular) and `scriptElems` (plural)